applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_stationary_solver.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, main_model_part, custom_settings):
         super(ConvectionDiffusionStationarySolver, self).__init__(main_model_part, custom_settings)
 
         # Overwrite the base solver minimum buffer size
-        if (self.settings["element_replace_settings"]["element_name"].GetString() == "EulerianConvDiff"):
+        if (self.settings["element_replace_settings"]["element_name"].GetString() in ["EulerianConvDiff","SymbolicQSConvectionDiffusionExplicit","SymbolicDConvectionDiffusionExplicit"]):
```

Comment:
You must not modify the stationary solver. Your element is not compatible with this. Please revert the changes.

Reply:
Done.